### PR TITLE
ED-1376: Publish apidoc and add eslint check

### PR DESCRIPTION
**Pull request by crystalynliu**

### CODE REVIEW

**Review by xsteadybcgo — APPROVED** [review]

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# http://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
```
Comment (BruceChar) [inline .editorconfig:7]: I think indent_size=4 is better, since it's not like front-end with so many nested layer

File: README.md [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
-## Api
+The Parallel service provides a basic interface (include both Heiko and Parallel chain data) that allows projects to aggregate data according to the needs of the scenario.
 
-You can look up all the endpoint documentation [here](./docs/api.md), its routing design follows the _RESTful_ specification.
+## Documents
 
-## Development
-
-### Dependencies:
+- [Api Documents](https://api-docs.parallel.fi/auction-service)
```
Comment (BruceChar) [inline README.md:7]: does this copy error?

File: README.md [hunk 3]
```diff
@@ -25,21 +23,26 @@ yarn install
...
+## Deployment
+
+### Actions
+
+- Staging: [Deployment action](https://github.com/parallel-finance/parallel-service/actions/workflows/staging-deployment.yml) would be triggered automatically when merging code to main branch
+- Production: Click `Run workflow` manually with main branch in [Deployment action](https://github.com/parallel-finance/parallel-service/actions/workflows/production-deployment.yml)
+
+## Dependencies:
+
+- [events-subql](https://github.com/parallel-finance/heiko-events-suqbl)
```
Comment (BruceChar) [inline README.md:48]: renamed: https://github.com/parallel-finance/parallel-events-suqbl.git, seems parallel has some ambiguity...it refers to parallel- finance not chain parallel

File: .editorconfig (hunk 1 above)
Comment (crystalynliu) [reply .editorconfig:7]: I think the suggestion for js is 2 space

File: README.md (hunk 2 above)
Comment (crystalynliu) [reply README.md:7]: Good catch

File: .editorconfig (hunk 1 above)
Comment (xsteadybcgo) [reply .editorconfig:7]: 2 is more commonly used

Comment (BruceChar) [reply .editorconfig:7]: Anyway! It's just strange to me 囧

**Review by BruceChar — APPROVED** [review]

File: packages/app/src/controller/asset/events.ts [hunk 4]
```diff
@@ -1,190 +1,195 @@
...
+  const { asset_id } = ctx.params
+  if (!isValidInteger(asset_id)) {
+    throw Resp.Fail(Code.Param_Invalid, `invalid asset id` as Msg)
+  }
+
+  const pageSize = Number((size as string) || 10)
+  const filter = `{
         module: {
-            equalTo: "assets"
+            equalTo: 'assets'
```
Comment (BruceChar) [inline packages/app/src/controller/asset/events.ts:150]: cannot modify

**Pull request merged**